### PR TITLE
Set log timings to milliseconds from microseconds

### PR DESCRIPTION
**Pull request by mbranniganexpensify**
@tylerkaraszewski 

Reference issue $ https://github.com/Expensify/Expensify/issues/70635

Method to locate fix locations:

- Searched for every use of "us" string
- Searched all SINFO,SWARN, and similar calls
- Ran full authTests suite with tail -f /var/log/syslog | grep '[0-9]us' and tail -f /var/log/syslog | grep '[0-9].us' to find other referenced timings. Discovered one in Server-Expensify (There is a matching PR there)

Note: This PR is one of two to address $ https://github.com/Expensify/Expensify/issues/70635

### CODE REVIEW

File: BedrockCommand.cpp [hunk 1]
```diff
@@ -218,20 +218,20 @@ void BedrockCommand::finalizeTimingInfo() {
...
+          << commitSyncTotal/STIME_US_PER_MS << ", "
+          << queueWorkerTotal/STIME_US_PER_MS << ", "
+          << queueSyncTotal/STIME_US_PER_MS << ", "
+          << totalTime/STIME_US_PER_MS << ", "
+          << unaccountedTime/STIME_US_PER_MS << ", "
+          << escalationTimeUS/STIME_US_PER_MS << ". Upstream: "
+          << upstreamPeekTime/STIME_US_PER_MS << ", "
+          << upstreamProcessTime/STIME_US_PER_MS << ", "
+          << upstreamTotalTime/STIME_US_PER_MS << ", "
+          << upstreamUnaccountedTime/STIME_US_PER_MS << "."
```
Comment (tylerkaraszewski) [inline BedrockCommand.cpp]: Same as in the other PR, I find these constants more confusing and difficult to remember than just using `1000`.

File: BedrockServer.cpp [hunk 2]
```diff
@@ -826,7 +826,7 @@ void BedrockServer::worker(SData& args,
                                 uint64_t preLockTime = STimeNow();
                                 shared_lock<decltype(server._syncThreadCommitMutex)> lock1(server._syncThreadCommitMutex);
                                 SINFO("_syncThreadCommitMutex acquired in worker in " << fixed << setprecision(2)
-                                      << ((STimeNow() - preLockTime)/1000.0) << "ms.");
+                                      << ((STimeNow() - preLockTime)/STIME_US_PER_MS) << "ms.");
```
Comment (tylerkaraszewski) [inline BedrockServer.cpp]: I think you misunderstood this line in particular (it logs decimal milliseconds already).

Comment (mbranniganexpensify) [reply BedrockServer.cpp]: I was just changing the extant 1000 to the constant "while I was there" for consistency. Now it's consistently annoying. :)

**Comment by mbranniganexpensify** [comment]
Removed STIME_US_PER_MS everywhere, which included some spots that were already in milliseconds.

**Pull request merged**